packages/studio-ui/src/web/views/FlowBuilder/diagram/nodes/SkillCallContents/index.tsx

```diff
@@ -21,6 +21,15 @@ type Props = Pick<BlockProps, 'node'>
 const SkillCallContents: FC<Props> = ({ node }) => {
   return (
     <div className={cx(style.contentsWrapper, style['skill-call'])}>
+      {node.previewElements?.map((item, i) => {
+        return (
+          <ActionItem
```

Comment:
is there any possibility that previewElements are not content element ?

Reply:
@EFF Not with the current implementation, but I left a comment above with another suggestion on how this additional use case can be implemented in the flow generator to preview an action, which I think can be useful for some use cases.

Let me know if you think about having that change to also allow actions